lib/postinstall.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2016 Digital Bazaar, Inc. All rights reserved.
+ */
+
+var async = require('async');
+var fs = require('fs');
+
+var parentPackage = require('../../../../package.json');
+
+async.auto({
+  createNodeSymlink: function(callback) {
+    fs.symlink('../../', '../' + parentPackage.name, 'dir', callback);
+  },
+  checkBowerJson: function(callback) {
+    // if an err is passed to callback here, we can stop this process
+    fs.stat('../../bower.json', callback);
+  },
+  checkComponents: ['checkBowerJson', function(callback) {
+    fs.stat('../../bower_components', function(err) {
+      if(err) {
+        return fs.mkdir('../../bower_components', callback);
+      }
+      callback();
+    });
+  }],
+  createBowerSymlink: ['checkComponents', function(callback) {
+    fs.symlink(
+      '../../', '../../bower_components/' + parentPackage.name, 'dir',
+      callback);
+  }]
```

Comment:
Would it be better to do bower stuff through the config system or via a symlink?